.envtemplate

```diff
@@ -0,0 +1,2 @@
+PORT=
```

Comment:
Let's append API_ to these to make sure they're component specific. We won't be doing any REACT_ one's but we don't know what the complete architecture looks like yet.

Reply:
Changed in all cases to API_PORT